conda_smithy/configure_feedstock.py

```diff
@@ -599,6 +599,7 @@ def _collapse_subpackage_variants(
         "channel_targets",
         "docker_image",
         "build_number_decrement",
+        "python_min",
```

Comment:
Let's add this only if `python_min` appears anywhere in the meta.yaml? (Just a text search)

Reply:
I was going on the fact that e.g.
https://github.com/conda-forge/conda-smithy/blob/6d9c019e5c1ed5e4916ef61ceb45e2fd38089617/conda_smithy/configure_feedstock.py#L602-L606
are added unconditionally as well, but we can make it conditional if you prefer.

Reply:
Though OTOH, in `_collapse_subpackage_variants` we don't seem to have easy access to the raw yaml string anymore, so we'd have to do something like
https://github.com/conda-forge/conda-smithy/blob/6d9c019e5c1ed5e4916ef61ceb45e2fd38089617/conda_smithy/configure_feedstock.py#L546
(only for `meta.yaml` obviously) plus extra handling for v1 recipes.

Reply:
See https://github.com/conda/conda-build/pull/5599 for `cdt_*`

Reply:
You can do `m.get_recipe_text()` right?

Reply:
> You can do `m.get_recipe_text()` right?

I tried that just now, but the meta for `lit` has already been skipped at that point. Oh well, off to reading the recipe I go.